due-processed/inc/masthead.php

```diff
@@ -0,0 +1,18 @@
+<?php
+use Carbon_Fields\Container;
+use Carbon_Fields\Field;
+
+function crb_attach_post_meta() {
+	Container::make( 'post_meta', __( 'Page Options' ) )
+		->where( 'post_id', '=', '7' ) // 7 is the id of the about page
```

Comment:
did you get a chance to look into whether we could target the article slug instead of the `id`?

Reply:
I don't think there is a way to target the article slug (there's no `page-about` class unfortunately). There is a `.page` class we could target, but then every page would have these styles.

Reply:
Could you target by `post_template`?
```suggestion
		->where( 'post_template', '=', 'page-about.php' ) // 7 is the id of the about page
```
Does that work? See [example](https://carbonfields.net/docs/guides-section-layout-2/?crb_version=2-2-0).

Reply:
Yes that works! Thanks

Reply:
I ended up just removing this check.